File: EMS/EMS/EMSSpeaker.m
//
//  EMSSpeaker.m
//

#import "EMSSpeaker.h"

@implementation EMSSpeaker

- (NSString *)description {
    NSMutableString *description = [NSMutableString stringWithFormat:@"<%@: ", NSStringFromClass([self class])];
    [description appendFormat:@"self.name=%@", self.name];
    [description appendFormat:@", self.href=%@", self.href];
    [description appendFormat:@", self.bio=%@", self.bio];
    [description appendFormat:@", self.thumbnailUrl=%@", self.thumbnailUrl];
    [description appendFormat:@", self.lastUpdated=%@", self.lastUpdated];
    [description appendString:@">"];
    return description;
}

@end

File: EMS/Network/Model/EMSSpeaker.swift
import Foundation

@objc class EMSSpeaker: NSObject {
    var name: String?

    var href: NSURL?

    var bio: String?

    var thumbnailUrl: NSURL?

    var lastUpdated: NSDate?

    override var description: String {
        return "<\(self.dynamicType): self.name=\(self.name), self.href=\(self.href), self.bio=\(self.bio), self.thumbnailUrl=\(self.thumbnailUrl), self.lastUpdated=\(self.lastUpdated)>"
    }
}